docs/PclExtension.md

```diff
@@ -0,0 +1,33 @@
+# RGL PCL extension
+
+The extension introduces the nodes and functions for point cloud processing. RGL features are enhanced with:
+
+- Downsampling
+- Writing to PCD file
+- Visualization
+
+For operations listed above, the extension uses [Point Cloud Library](https://pointclouds.org/).
+
+## Building
+
+Before building RGL PCL extension, it is necessary to install the required dependencies. Run the setup script with `--install-pcl-deps` flag to download and install them. It could take some time (Point Cloud Library is built from source):
```

Comment:
```suggestion
After building RGL PCL extension, it is necessary to install the required dependencies. Run the setup script with `--install-pcl-deps` flag to download and install them. It could take some time (Point Cloud Library is built from source):
```